assets/src/edit-story/utils/nativeCopyPasteExpected.js

```diff
@@ -29,7 +29,7 @@ function nativeCopyPasteExpected() {
     'true' === contentEditable ||
     'textarea' === tagName.toLowerCase() ||
     ('input' === tagName.toLowerCase() &&
-      ('text' === type || 'number' === type))
+      ['text', 'number', 'search', 'email', 'tel', 'url'].includes(type))
```

Comment:
Ah, it was the type, that was the trigger. I thought it was the dialog and the portal to a different section of the document... clever!